app/models/mixins/relationship_mixin.rb

```diff
@@ -44,10 +44,11 @@ def reload(*args)
     super
   end
 
+  # only used from specs
   def clear_relationships_cache(*args)
     options = args.extract_options!
-    to_clear = options[:only] ? options[:only].to_miq_a : (RelationshipMixin::MEMOIZED_METHODS - options[:except].to_miq_a)
```

Comment:
@kbrock `options[:only]` code was deleted here, which has nothing to do with Rubocop.